react.js

```diff
@@ -14,6 +14,25 @@ module.exports = {
     }
   },
   rules: {
+    // Turn off the following rules to prevent conflicts when used with Prettier
+
+    /*
+      We recommend the following config if not using Prettier:
```

Comment:
For react and typescript, since we can't add these rules to the general `stylistic` config, I kept the overrides in their respective files and assumed Prettier is on. I provided recommendations to use if the consumer is not using Prettier. It's not _amazing_ but the best I could come up with.

cc: @mttrms

Reply:
I think it's a fair assumption to make and I'm totally fine with this